docs/book/src/managed/managedcluster.md

```diff
@@ -331,11 +331,7 @@ metadata:
 spec:
   extensions:
   - name: my-extension
-    extensionType: "TraefikLabs.TraefikProxy"
-    plan:
-      name: "traefik-proxy"
-      product: "traefik-proxy"
-      publisher: "containous"
+    extensionType: "microsoft.flux"
```

Comment:
@alimaazamat I'll close this PR in favor of yours, could you make one more update to your PR to include the doc updates so we no longer reference this removed extension?

thank you!